### DISCUSSION TITLE
match appstore hangs on `Cloning remote git repo...`

### DISCUSSION THREAD

**Discussion by RoyDeng** [fastlane/fastlane#20024] [Q&A]
GitLab, which was used by my company, added SSH. According to the practice of most people, I added the key to the formula, and the final process will be stuck here.

```
match(
  type: "appstore", 
  api_key: api_key,
  app_identifier: "xxx.xxx.xxx.xxx",
  readonly: true,
  verbose: true,
  keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
  keychain_password: ENV["MATCH_KEYCHAIN_PASSWORD"],
  shallow_clone: true,
  git_private_key: "/Users/test/.ssh/xxxxxx"
)
```

fastlane

- version: 2.204.2

**Comment by lucgrabowski** [comment]
Try running git clone manually on the machine you use. When I had similar issues, I simply ran `git clone` manually in terminal, confirmed that I allow the connection and it started working fine.

**Reply by RoyDeng** [reply]
It's working when using clone manually on the machine.

```
$ ssh-agent bash -c 'ssh-add /Users/xxxxxx/.ssh/id_xxxxxx; git clone https://xxx.xxx.xxx/xxx/certificates.git /var/folders/pf/xxxxxx/T/xxxxxx --depth 1 --no-single-branch'
```

**Comment by Dante1349** [comment]
this issue still persists, please help: https://github.com/fastlane/fastlane/issues/21826
